services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -460,6 +465,9 @@ protected void startServices() {
         blobSidecar ->
             recentBlobSidecarsFetcher.cancelRecentBlobSidecarRequest(
                 new BlobIdentifier(blobSidecar.getBlockRoot(), blobSidecar.getIndex())));
+    executionProofManager.subscribeToValidExecutionProofs(
+        (executionProof, remoteOrigin) ->
+            LOG.debug("Received valid execution proof: {}", executionProof));
```

Comment:
This is just temporary until we hook up the validation logic? If that is the case it is probably a good idea to add a TODO.